docs/config/lua/keyassignment/QuickSelectArgs.md

```diff
@@ -31,6 +31,7 @@ The `QuickSelectArgs` struct allows for the following fields:
 * `patterns` - if present, completely overrides the normal set of patterns and uses only the patterns specified
 * `alphabet` - if present, this alphabet is used instead of [quick_select_alphabet](../config/quick_select_alphabet.md)
 * `action` - if present, this key assignment action is performed as if by [window:perform_action](../window/perform_action.md) when an item is selected.  The normal clipboard action is NOT performed in this case.
+* `skip_action_on_paste` - overrides whether `action` is performed after an item is selected using a capital value (when paste occurs).
```

Comment:
```suggestion
* `skip_action_on_paste` - overrides whether `action` is performed after an item is selected using a capital value (when paste occurs). {{since('nightly', inline=True)}}
```